tests/acceptance/pageObjects/filesPage.js

```diff
@@ -257,10 +257,12 @@ module.exports = {
       })
     },
     confirmFileOverwrite: function () {
-      return this
-        .waitForElementVisible('@fileOverwriteConfirm')
-        .click('@fileOverwriteConfirm')
-        .waitForElementNotVisible('@fileOverwriteConfirm')
+      return this.waitForElementVisible('@fileOverwriteConfirm')
+        .api.moveToElement(this.elements.fileOverwriteDialog.selector, 0, 0)
+        .mouseButtonClick(0)
+        .waitForElementNotVisible(this.elements.newFolderButton.selector)
+        .click(this.elements.fileOverwriteConfirm.selector)
```

Comment:
Why click again? also what is different between click and mouseButtonClick? Does this fail if the first click works and the dialog disappears?

Reply:
See the comment above.